api/src/pcapi/routes/e2e/routes/sandbox_data.py

```diff
@@ -0,0 +1,39 @@
+from flask import current_app as app
+
+from pcapi.core.finance import factories as finance_factories
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offerers.models import VenueTypeCode
+from pcapi.core.users import factories as users_factories
+from pcapi.repository.clean_database import clean_all_database
+from pcapi.sandboxes.scripts.mocks.educational_siren_mocks import MOCK_ADAGE_ELIGIBLE_SIREN
+from pcapi.serialization.decorator import spectree_serialize
+
+
+@app.route("/e2e/pro/create-event-individual-offer", methods=["GET"])
+@spectree_serialize(on_success_status=204)
+def setup_create_event_individual_offer() -> None:
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏J'ai l'impression que le nom nous ment ici

On dit "je te crée une offre individuelle de type event"
On fait "Créer un pro avec un lieu et tout le stuff attaché"

Reply:
Ah, en fait pour moi c'était setup le test E2E appelé: create event individual offer